tcms/testruns/views.py

```diff
@@ -63,57 +72,66 @@ def post(self, request):
                                                'category',
                                                'priority').order_by('pk')
 
-        if request.POST.get('POSTING_TO_CREATE'):
-            form = NewRunForm(request.POST)
-            form.populate(product_id=test_plan.product_id)
-
-            if form.is_valid():
-                # Process the data in form.cleaned_data
-                default_tester = form.cleaned_data['default_tester']
-
-                test_run = TestRun.objects.create(
-                    product_version=test_plan.product_version,
-                    stop_date=None,
-                    summary=form.cleaned_data.get('summary'),
-                    notes=form.cleaned_data.get('notes'),
-                    plan=test_plan,
-                    build=form.cleaned_data['build'],
-                    manager=form.cleaned_data['manager'],
-                    default_tester=default_tester,
-                )
-
-                loop = 1
-                for case in form.cleaned_data['case']:
-                    try:
-                        tcp = TestCasePlan.objects.get(plan=test_plan, case=case)
-                        sortkey = tcp.sortkey
-                    except ObjectDoesNotExist:
-                        sortkey = loop * 10
-
-                    test_run.add_case_run(case=case, sortkey=sortkey,
-                                          assignee=default_tester)
-                    loop += 1
-
-                return HttpResponseRedirect(
-                    reverse('testruns-get', args=[test_run.pk, ])
-                )
-
-        else:
-            form = NewRunForm(initial={
-                'summary': 'Test run for %s' % test_plan.name,
-                'manager': test_plan.author.email,
-                'default_tester': request.user.email,
-                'notes': '',
-            })
-            form.populate(product_id=test_plan.product_id)
+        form = NewRunForm(initial={
+            'summary': 'Test run for %s' % test_plan.name,
+            'manager': test_plan.author.email,
+            'default_tester': request.user.email,
+            'notes': '',
+        })
+        form.populate(product_id=test_plan.product_id)
 
         context_data = {
             'test_plan': test_plan,
             'test_cases': tcs_values,
             'form': form,
+            'disabled_cases': disabled_cases
```

Comment:
self note: I think this will fix #718 but need to inspect it more closely once everything else here is done.

Reply:
Confirmed, this resolves #718 so add that to the commit log.

Reply:
:arrow_double_up: